docs/api/go/getting-started.md

```diff
@@ -0,0 +1,109 @@
+# Get started
+
+In this guide we will demonstrate how to:
+
+1. Connect to the zkSync network.
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of go and you have it configured on your local machine.
```

Comment:
"you are familiar with the Go programming language", including a link to the Go documentation